style.css

```diff
@@ -0,0 +1,452 @@
+*{
+    box-sizing: border-box
+}
+
+body{
+    font-family: Helvetica, Arial, sans-serif;
+    line-height: 20px;
+    margin: 0;
+    padding: 0;
+}
+
+html{
+    scroll-behavior: smooth;
+}
+
+/*GLOBAL*/
+
+.container{
+    width: 90%;
+    margin: 0 auto;
+}
+
+.invert{
+    filter: invert(100%);
+}
+
+/*HEADER*/
+
+header{
+    height: 150px;
+}
+
+header li a{
+    color: #565254;
+}
+
+.navBar ul {
+    list-style: none;
+    margin: 0;
+    padding: 0;
+}
+
+.navBar li{
+    display: inline-block;
+    padding: 10px;
+    margin: 10px;
+    border: 2px solid transparent;
+}
+.narBar .more{
+    border: 2px solid black;
+}
+
+.navBar li:hover{
+    border: 2px solid #565254;
+}
+
+.navBar li a:visited{
+    color: #565254;
+}
+
+.navBar .navHidden{
+    display: none;
+}
+/*FEATURE PAGE*/
+
+.featureImage {
+    text-align: center;
+    height: 1100px;
+    background-image: url(images/featureimage.jpg);
+    background-size: cover;
+    background-position: center;
+}
+
+.featureImage h1{
+    letter-spacing: 25px;
+    line-height: 5rem;
+    font-size: 64px;
+    padding-top: 240px;
+    margin-bottom: 30px;
+    color: #565254;
+    font-family: Verdana, Geneva, sans-serif;
+    font-weight: lighter;
+}
+
+.featureImage p{
+    letter-spacing: 18px;
+    font-size: 40px;
+    color: #FFFBFE;
+    font-family: Verdana, Geneva, sans-serif;
+}
+
+.featureImage a{
+    text-decoration: none;
+    font-size: 18px;
+    letter-spacing: 5px;
+    margin: 5px 10px;
```

Comment:
This line does nothing.

Reply:
Its to do with the transparent border on the navbar that turns on when you hover.